test/to_do_app_web/controllers/task_controller_test.exs

```diff
@@ -0,0 +1,84 @@
+defmodule ToDoAppWeb.TaskControllerTest do
```

Comment:
Run this test and ensure that it is working , even if you make changes in the files elsewhere so that the test suite doesn't break.